config/locales/misc/fr.yml

```diff
@@ -31,6 +31,7 @@ fr:
     application_policy:
       must_be_logged_in: must be logged in
       you_must_be_logged_in: You must be logged in
+      your_account_is_suspended: Your account is suspended
```

Comment:
Reading note, this is not yet used as I have a follow-up PR that will use this.  But it makes some sense to include.